libs/sync/src/lib/sync-details-page/sync-details-page.component.html

```diff
@@ -18,13 +18,15 @@ <h1 class="page-title">
               <pa-toggle
                 withBackground
                 [value]="!sync.disabled"
+                qa="active-toggle"
```

Comment:
following what we learned while implementing the e2e tests, using custom attributes like `qa` is not a good practice. Why not using the same as what we use for cypress: `data-cy` ? this way we could reuse them in e2e tests at some point if needed.